Packs/MicrosoftGraphListener/ReleaseNotes/1_0_21.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### O365 Outlook Mail Single User (Using Graph API)
+- Fixed an issue where inline images were not getting into attachments list.
```

Comment:
Lets be more specific here - please mention the scope when this happens -command/use case